### ISSUE TITLE
Several dialogs have [Escape] for yes instead of no (defaults)

### ISSUE THREAD

**Issue by wally-crunk** [transmission-remote-gui/transgui#1450]
1. Mac OS Ventura 13.3 (22E252)
2. Transmission Daemon 4.0.2 (2a57b17031)
3. Transmission GUI 5.18 https://github.com/transmission-remote-gui/transgui/commit/3fb1d339f600d9d973aa9b923b548a369bd3adbf

Several of the Warning dialog boxes in Transmission-GUI have `Escape` key mapped to Yes instead of No, which is definitely not the expected behavior.

For example,
<img width="372" alt="Screenshot to verify " src="https://user-images.githubusercontent.com/122107861/229162620-4da2288f-1b30-49b6-a91e-36ad16a50e3d.png">
[Escape] causes Verification to start.

For example,
<img width="372" alt="Screenshot to remove" src="https://user-images.githubusercontent.com/122107861/229163266-acd55d57-56fd-45dc-b833-b98908667fc2.png">
[Escape] causes the Data to be deleted -- cancel or "No" is what would be expected.

(https://github.com/transmission-remote-gui/transgui/issues/1382)

**Comment by pwinn** [comment]
Yes, this is horrifying, and has resulted in unwanted data loss.

**Comment by wally-crunk** [comment]
@pwinn I believe @xavery is addressing this issue in their fork.

**Comment by xavery** [comment]
@wally-crunk I don't think this particular thing is fixed yet, though I will add an issue so I can at least track this.

**Comment by pwinn** [comment]
@wally-crunk I switched to that fork a few days ago, and you're right!

@xavery I tested this in v5.18.3.f build b036107, and it works as expected: Right-click, Remove torrent and data, then Esc closes the confirmation dialog and leaves the torrent in place. Whew!

**Comment by xavery** [comment]
@pwinn That's great to know! I don't remember doing anything to address this specifically so must've been something that got fixed in Lazarus in the meantime.
